src/pybind/mgr/dashboard/controllers/summary.py

```diff
@@ -64,9 +64,10 @@ def _task_permissions(self, name):
         return result
 
     def _get_host(self):
-        mgr_map = mgr.get('mgr_map')
-        services = mgr_map['services']
-        return services['dashboard']
+        services = mgr.get('mgr_map')['services']
```

Comment:
If possible please add
```suggestion
	    # type: () -> dict
        services = mgr.get('mgr_map')['services']
```